shared/kubernetes/consts.go

```diff
@@ -17,3 +17,9 @@ const (
 	PersistentVolumeClaimName  = MizuResourcesPrefix + "volume-claim"
 	MinKubernetesServerVersion = "1.16.0"
 )
+
+const (
+	MizuLabelPrefix         = "mizu.io/"
+	LabelMizuManagement     = MizuLabelPrefix + "management-method"
```

Comment:
an idea we should consider, as we planing to support mizu Ent. install with helm, we should use the same label:
"managed-by".
the value can be something like "mizu-cli", when mizu cli created to resources, and responsible of cleaning them